src/foolscap/broker.py

```diff
@@ -174,21 +174,21 @@ def initBroker(self):
 
         # tracking Referenceables
         # sending side uses these
-        self.nextCLID = count(1).next # 0 is for the broker
+        self.nextCLID = count(1) # 0 is for the broker
```

Comment:
A smaller change with less chance of introducing backwards incompatibility would be to make the line:

```
self.nextCLID = functools.partial(six.advance_iterator, count(1))
```

This lets all of the callers remain unchanged and seems to preserve the apparent interface (which is "call it with no arguments").

Reply:
I am not a huge fan of `partial` as it changes the type of the object - wrapping everything up in partial and sometimes making debugging difficult. I think it often produces `too clever` code. Anyway made the change. Thanks.